app/helpers/generic_work_helper.rb

```diff
@@ -5,6 +5,6 @@ def render_collection_links(solr_doc)
     links = collection_list.map do |collection|
       link_to collection.title_or_label, collection_path(collection.id)
     end
-    content_tag :span, t(:is_part_of) + ': ' + links.join(', ').html_safe
+    content_tag :span, t('sufia.collection.is_part_of') + ': ' + links.join(', ')
```

Comment:
I think that `html_safe` was required. That's why the test changed below. Can you please add that back or use [safe_join](http://api.rubyonrails.org/classes/ActionView/Helpers/OutputSafetyHelper.html#method-i-safe_join)?